test-it/karma/utilities/TestHelpers.VireoRunner.js

```diff
@@ -84,13 +89,29 @@
                     return result;
                 }
             };
+        },
+        toMatchIEEE754Number: function (util, customEqualityTesters) {
+            return {
+                compare: function (actual, expected) {
+                    var result = {};
+                    if (typeof actual === 'number' && typeof expected === 'number') {
+                        result.pass = Object.is(actual, expected);
```

Comment:
Does "Object.is" expect the exact same value for floating point numbers? If so, we should use that only for special floating point numbers (such as NaN, Inf, +0, etc) and then use some epsilon comparison for regular floating point numbers. I wonder if we can use toBeCloseTo matcher in Jasmine?